src/js/tracks/text-track.js

```diff
@@ -20,6 +20,14 @@ import XHR from 'xhr';
  * @param {Track} track track to addcues to
  */
 const parseCues = function(srcContent, track) {
+  // Make sure that vttjs has loaded, otherwise, wait till it finished loading
+  // NOTE: this is only used for the alt/video.novtt.js build
+  if (typeof window.WebVTT !== 'function') {
```

Comment:
Should this block `return;`?

Otherwise line 31 [https://github.com/videojs/video.js/pull/3181/files#diff-45ae6eaab623b37642b347fe7707682aR31] will fire an exception.

![screen shot 2016-03-15 at 2 06 54 pm](https://cloud.githubusercontent.com/assets/549319/13794101/74d3074a-eab7-11e5-94d7-a34cbcac4fb2.png)